components/Header/Header.tsx

```diff
@@ -48,21 +66,45 @@ export function Header() {
         </HomeLinkAndPageDescriptionWrapper>
         <WalletAndMenuWrapper>
           {showDelegationNotification && (
-            <DelegationNotificationWrapper style={delegationNotificationStyle}>
+            <NotificationWrapper style={delegationNotificationStyle}>
```

Comment:
this notification lets the user know that they have v1 rewards. clicking it opens the claim panel